packages/console/src/components/Table/TableEmpty.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import EmptyImage from '@/assets/images/table-empty.svg';
+
+import * as styles from './TableEmpty.module.scss';
+
+type Props = {
+  title?: string;
+  content?: string;
+  children?: ReactNode;
+};
+
+const TableEmpty = ({ title, content, children }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <tr>
+      <td colSpan={0}>
+        <div className={styles.tableEmpty}>
+          <div className={styles.image}>
+            <img src={EmptyImage} />
+          </div>
+          <div className={styles.title}>{title ?? t('admin_console.errors.empty')}</div>
```

Comment:
nit: {title && <div className={styles.title}>{ t('admin_console.errors.empty')}</div>} ?

Reply:
Is a default title, `&&` may not work?